OpenRA.Mods.RA/Traits/Infiltration/Infiltrates.cs

```diff
@@ -58,6 +58,10 @@ bool IsValidOrder(Actor self, Order order)
 			if (order.ExtraData == 0 && order.TargetActor == null)
 				return false;
 
+			// Should not infiltrate own or allied
+			if (order.TargetActor.Owner.IsAlliedWith(self.Owner))
```

Comment:
I think he means that if you have a field in the Info (set to not allow infiltrating allies by default, and none of our mods changing the default, ofc) some thirdparty mod can change it if they like, or custom maps can still enable it for their crazy gameplay.